cli/js/lib.deno.ns.d.ts

```diff
@@ -778,6 +778,10 @@ declare namespace Deno {
     /** String that should follow the random portion of the temporary
      * directory's name. */
     suffix?: string;
+    /** Permissions to use when creating the file or directory (defaults to
+     * `0o700`, before the process's umask).
+     * Does nothing/raises on Windows. */
```

Comment:
Does it do nothing or does it raise on Windows? Could you clarify the comment?

Reply:
I hoped to prompt you (or others on the dev team) to weigh in on what you wanted to do in such cases. Noop or throw? Somewhere I have a list of all the functionality I came across where this choice needed to be made; I'll look for it and make an issue. It'd be good to have an across-the-board decision. My inclination would be to throw.

Reply:
I think in this case no-op would be best bet...